pydoctor/test/epydoc/test_epytext2html.py

```diff
@@ -152,15 +142,38 @@ def test_epytext_sections() -> None:
           This is a paragraph in section 2.
         '''
     expected = '''
-        <p>This paragraph is not in any section.</p>
-        <h1>Section 1</h1>
-        <p>This is a paragraph in section 1.</p>
-        <h2>Section 1.1</h2>
-        <p>This is a paragraph in section 1.1.</p>
-        <h1>Section 2</h1>
-        <p>This is a paragraph in section 2.</p>
+
+          <p>
+           This paragraph is not in any section.
+          </p>
+          <div class="rst-section">
+           <h2 class="heading">
+            Section 1
+           </h2>
+           <p>
+            This is a paragraph in section 1.
+           </p>
+           <div class="rst-section">
+            <h3 class="heading">
+             Section 1.1
+            </h3>
+            <p>
+             This is a paragraph in section 1.1.
+            </p>
+           </div>
+          </div>
+          <div class="rst-section">
+           <h2 class="heading">
+            Section 2
+           </h2>
+           <p>
+            This is a paragraph in section 2.
+           </p>
+          </div>
+
         '''
-    assert epytext2html(doc) == squash(expected)
```

Comment:
I'll resurrect this `squash` function since the same code is called again and again...

Reply:
Done.